avro2tf/src/main/scala/com/linkedin/avro2tf/parsers/TensorizeInJobParamsParser.scala

```diff
@@ -172,9 +176,18 @@ object TensorizeInJobParamsParser {
       .action(
         (tensorizeInConfigPath, tensorizeInParams) => {
 
-          val bufferedSource = Source.fromFile(tensorizeInConfigPath)
-          val jsonInput = bufferedSource.mkString
-          bufferedSource.close()
+          var jsonInput = ""
+
+          // Read JSON config from local if exists, otherwise read from HDFS
+          if (new File(tensorizeInConfigPath).exists()) {
+            val bufferedSource = Source.fromFile(tensorizeInConfigPath)
+            jsonInput = bufferedSource.mkString
+            bufferedSource.close()
+          } else {
+            val fs = FileSystem.get(new JobConf())
+            jsonInput = IOUtils.readContentFromHDFS(fs, new Path(tensorizeInConfigPath))
+            fs.close()
+          }
```

Comment:
Do we want to throw an exception message (to maybe inform the user...) if neither of them exists?